src/app/_helpers/resources.interceptor.ts

```diff
@@ -1,73 +1,49 @@
 import { Observable } from 'rxjs';
 import { Injectable } from '@angular/core';
-import { tap } from 'rxjs/operators';
 import {
-  HttpRequest,
+HttpRequest,
   HttpHandler,
   HttpEvent,
   HttpInterceptor,
-  HttpResponse,
-  HttpErrorResponse
+  HttpResponse
 } from '@angular/common/http';
 
-// const resource: Resources[] = [
-//   {amount: 500,
-//     type: 'food',
-//     generation: 0 },
-
-//    {amount: 320,
-//     type: 'diamond',
-//     generation: 1 }
-// ];
-
 @Injectable()
 export class ResourceInterceptor implements HttpInterceptor {
   constructor() {}
   // function which will be called for all http calls
   intercept(
     request: HttpRequest<any>,
     next: HttpHandler
-  ):
-   Observable<HttpEvent<any>> {
+  ): Observable<HttpEvent<any>> {
     // how to update the request Parameters
 
-    if (request.url.endsWith('/kingdom/resources') &&
-    (request.method === 'GET'))  {
+    if (request.url.endsWith('/game/resources')) {
       return new Observable(observable => {
-        observable.next(new HttpResponse<any>(
-          {
-            body:
-            {
-              amount: 500,
-              type: 'food',
-              generation: 0
-            },
-            status: 200
-          }
-        ));
+        observable.next(
+          new HttpResponse<any>({
+            body: {
+              resources: [
+                {
+                  amount: 500,
+                  type: 'food',
+                  generation: 0
+                }
+              ],
+              resources2: [
+                {
+                  amount: 340,
+                  type: 'money',
+                  generation: 1
+                }
+              ],
```

Comment:
@SamHerrera88 this is not how the backend will respond, it will give you one resources array, containing 2 objects, for `food` resources and `gold` resources